Installing-and-Configuring/OS/index.md

```diff
@@ -79,17 +79,34 @@ This feature may be turned off during configuration of heads.  The following wil
 
 ### Using only a LUKS passphrase
 
-In this model heads is not responsible for any of the encryption of data on disk.  The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase.  The correct passphrases allows boot to finish.
+In this model heads is not responsible for any of the decryption of data on disk.  The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase.  The correct passphrases allows boot to finish.
 
-This model is vulnerable to brute force attacks but does not lock the data to the hardware.
+This model is vulnerable to brute force attacks.
 
-### Security Dongle (aka smartcard aka security token)
+### Security Dongle with LUKS root
 
-You may use a security dongle to manage the passphrase for the root LUKS partition.  This is also out of scope for heads even if using the same security dongle for verification.
+In this model heads is not responsible for any of the decryption of data on disk.  You may use a security dongle (aka smartcard aka security token) to manage the passphrase for the root LUKS partition.
 
-The first advantage to this approach is that the user only needs to type a PIN on boot rather than a passphrase.  Unlike the [heads TPM](#using-tpm-for-decryption-of-drives) model this model does not lock the data on disk to the hardware.  The hard drive may be moved to a different system and used with the security dongle and/or a LUKS passphrase.  Also, the security dongle enforces limits which prevent brute force attacks.
+The first difference in this approach compared to the [heads TPM](#using-tpm-for-decryption-of-drives) model is that the user may be limited to typing ONLY a short number sequence (PIN) rather than a full passphrase depending on smartcard hardware used.  The hard drive may be moved to a different system and used with the security dongle and/or a LUKS passphrase without modification.  Like the TPM, the security dongle enforces limits which prevent brute force attacks.
```

Comment:
Sorry. A PIN is not consensual. Namely, and basically, it simply means a short passphrase/password. For the rest of this, I will leave this unreviewed until @mrchromebox @kylerankin review. As per previous reviews, we both agreed this wiki should not be the place to document this. But you seem to want this in this PR, for which I want to encourage you to continue documenting, while seeing this process really involving and not wanting to do this too often.

@kylerankin please comment.

Reply:
This may be moot given comments below.  The librem key limits pin to 8 numbers.

Reply:
I removed this sentence.

Reply:
I confirm that my PIN is definitely more then 8 characters, not only numbers.

You added new stuff...
Can we chat somewhere to address this pull request correctly without adding another roll of 10 commits before we agree on content? Slack is where the community happens. Linked in doc.

Reply:
I edited based on your comment that the information was out of scope.  This and the conversation about something being 'WRONG' are the last two items.  It was not clear in your comments what  you wanted me to change.  The other comments suggested exact text and I updated to match.

I don't think that we are online at the same time so using Slack may not help.  You have mentioned multiple tools as alternatives to discussing this work in a PR.  I am committed to getting these docs updated and will use whatever tool is necessary.  I will have time tomorrow to try slack.

Once we agree on the copy I can create smaller PRs for various pages but this PR overhauls much of the documentation and pages link to each other so 'smaller' is challenging.  I can start the process over with a new feature branches to get smaller changes.  Perhaps broken down into these (but the links will be broken or I have to change them in multiple locations and try to make sure the last commit puts them all back):
* Operating Systems index and sub pages
* Step 3 - Configuring Keys
* Recovery Shell
* Upgrading heads
* Prereq
* Other?

Another option is that I start a new feature branch and squash all of the changes into 1 commit under my name and start a new PR.  That would reduce noise for the actual commit and this PR is available for searching and history of discussion.